packages/grpc-js/src/server.ts

```diff
@@ -165,6 +170,10 @@ export class Server {
     if (this.channelzEnabled) {
       this.channelzTrace.addTrace('CT_INFO', 'Server created');
     }
+
+    this._channelzHandler = this._channelzHandler.bind(this)
+    this._streamHandler = this._streamHandler.bind(this)
```

Comment:
I don't like this. There's a kind of spooky action at a distance here, where the references to these functions in `_setupHandlers` work differently than normal because these lines are here. I would prefer to do this explicitly at that point where these functions need to be referenced.